sys/mips/mediatek/mtk_soc.c

```diff
@@ -54,6 +54,7 @@ static uint32_t mtk_soc_timerclk = MTK_CPU_CLK_880MHZ / 2;
 
 static uint32_t mtk_soc_chipid0_3 = MTK_UNKNOWN_CHIPID0_3;
 static uint32_t mtk_soc_chipid4_7 = MTK_UNKNOWN_CHIPID4_7;
+static char mtk_soc_cpu_model[64];
```

Comment:
All the changes in this file aren't needed.
However, #include <machine/md_var.h> is needed to pickup the cpu_model definition there.